server/routerlicious/packages/lambdas/src/scribe/lambda.ts

```diff
@@ -347,7 +348,9 @@ export class ScribeLambda implements IPartitionLambda {
 						`${value.operation.minimumSequenceNumber} != ${value.operation.sequenceNumber}`,
 					);
 					this.noActiveClients = true;
-					if (this.serviceConfiguration.scribe.generateServiceSummary) {
+                    const isTransientTenant = new Set(this.transientTenants).has(this.tenantId);
```

Comment:
Instead of computing the set from the string array every time, wouldn't it be better to keep the set as another variable, or possibly use the lambda factory to pass the set of tenants, instead of a string array?